andela/mmdp_site
# MMDP-336 Implement search for repository documents

**What does this PR do?**

This PR implements search functionality for documents under the repository section of the MMDP website.

**Description of Task to be completed?**

As an administrator, I want to be able to search for a document displayed in the 'Repository > Document' section of the 'Resources' page by entering search criteria in a search box.

**How should this be manually tested?**

Git clone https://github.com/andela/mmdp_site
Check out  ft-search-repository-documents-MMDP-336 branch using git.
Run yarn run: server to start the server.

**Any background context you want to provide?**

None

**What are the relevant Jira issues?**

[MMDP-336](https://viisaus.atlassian.net/browse/MMDP-336)
